src/fmgc/src/flightplanning/FlightPlanManager.ts

```diff
@@ -43,6 +44,17 @@ export enum FlightPlans {
     Temporary,
 }
 
+export enum FlightArea {
```

Comment:
I'd add a doc comment here describing what this is and what IRL property this maps to